### PR TITLE
add  "Hyperledger Besu Mode Grafana Dashboard" stats monitor

### PR DESCRIPTION
**Pull request by DRIgnazGortngschirl**

### CODE REVIEW

**Review by TheEnthusiasticAs — APPROVED** [review]

**Review by gitr0n1n — APPROVED** [review]
Nice. I added a forward to this page at: EthClassic.net

**Comment by DRIgnazGortngschirl** [comment]
> 
> 
> Nice. I added a forward to this page at: EthClassic.net

Seems that the site has no SSL cert. Can you update it? Users will see a warning.
image

**Review by ghost — APPROVED** [review]

**Comment by ghost** [comment]
Brave browser gives no warning, makes a smooth forward and shows a secure connection.

**Comment by DRIgnazGortngschirl** [comment]
> 
> 
> Brave browser gives no warning, makes a smooth forward and shows a secure connection.

Seems that Brave doesn't care about the SSL cert of the redirector site. Port 443 is not open at EthClassic.net. Mozilla 92.0.1 does with default settings. 

Anyways big thanks @gitr0n1n for adding this redirect please if you or someone else would like to help and/or improve this site any suggestions is very welcome!

**Comment by stevanlohja** [comment]
@MrMarioMichel Can you adjust your PR to commit to the `feature` branch instead of `master`?
Ref: https://github.com/ethereumclassic/Awesome-ETC/blob/master/Contributing.md

**Comment by DRIgnazGortngschirl** [comment]
@stevanlohja Done

**Comment by DRIgnazGortngschirl** [comment]
@gitr0n1n if you wanna you can also use this IP for the domain to get forwarded to my HTTP proxy will forward the client if coming with the right subdomain grafana.mariomichel.com. We could also fully migrate it to EthClassic.net your decision.

**Comment by stevanlohja** [comment]
@MrMarioMichel can you resolve conflicts? Is the scope of the change simply adding the `https://grafana.mariomichel.com/d/5S-6O8VZk/hyperledger-besu-node-at-ethereum-classic` link?

**Comment by DRIgnazGortngschirl** [comment]
@stevanlohja Yes but sadly this is a whole line which seems to be updated.

**Review by ghost — APPROVED** [review]

**Comment by stevanlohja** [comment]
No worries let me resolve it.

**Review by stevanlohja — APPROVED** [review]
Resolved the merge conflict, previewed the doc, tested new link LGTM. Thanks for running this dashboard :)

**Review by ghost — COMMENTED** [review]
Should it be 'Metrics' or is 'Metris' correct here?
(not a fan of a space behind '/' but that's just personal taste...)

File: README.md [hunk 1]
```diff
@@ -34,6 +34,8 @@ An awesome list of resources for the Ethereum Classic (ETC) project. This is an
     - [Emerald Receipt](https://receipt.emeraldpay.io/)
     - [ETCBlockExplorer](https://etcblockexplorer.com/)
     - [Expedition](https://expedition.dev/?network=mainnet)
+  - Metris/ Dashboards
```
Comment (stevanlohja) [inline README.md]:
```suggestion
  - Metrics/ Dashboards
```

**Comment by stevanlohja** [comment]
@w1g0 good catch! Fixed the spelling.

**Review by developerkevin — COMMENTED** [review]
lgtm - other than the dead or nonexistent broken links.

**Pull request merged**